core/listings.go

```diff
@@ -1219,10 +1219,11 @@ func validatePhysicalListing(listing *pb.Listing) error {
 		for _, region := range shippingOption.Regions {
 			if int(region) == 0 {
 				return errors.New("Shipping region cannot be NA")
-			} else if int(region) > 247 && int(region) != 500 {
-				return errors.New("Invalid shipping region")
+			} else {
```

Comment:
The logic before and after has changed. The logic before seems to error on country codes between 501-508 inclusive, whereas now those values would be ok. I would prefer to maintain existing logic within this PR. How about something like:

Some pseudo:
`if ok := validShippingRegion(region); !ok { ...`
```
func validShippingRegion(region) bool {
  return EnumValueMapLookupIsOk(region) && RegionIsNotAContinent(region)
}
```

Reply:
I wouldn't even be upset if you pulled in the `int(region) == 0` check in there, too. Maybe return a `(bool, error)` instead?